build.zig

```diff
@@ -98,7 +139,11 @@ pub fn build(b: *std.Build) !void {
         }
         if (b.lazyDependency("font_assets", .{})) |dep| module.addImport("font-assets", dep.module("font-assets"));
 
-        if (want_examples) try buildExamples(b, optimize, target, module);
+        if (want_examples) {
+            for (examples) |example| {
+                b.getInstallStep().dependOn(example.install_step);
+            }
```

Comment:
nit: prefer reducing this to one or two expressions:

```
if (want_examples) {
    for (examples) |example| b.getInstallStep().dependOn(example.install_step);
}
```

or

```
if (want_examples) for (examples) |example| b.getInstallStep().dependOn(example.install_step);
```

I usually decide based on line length exceeding >100 char